app/models/PageContextTargeting.scala

```diff
@@ -0,0 +1,8 @@
+package models
+
+case class PageContextTargeting(
```

Comment:
I have not updated the epic model to use this type as it already has the fields (`tagIds` etc) at the top level, so it'd require a data migration.
But I thought it was nice to put these fields together in a type that can be passed around in SAC + SDC